spec/models/note_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe Note, type: :model do
+  @user = User.new
+  note = @user.notes.new(title: "Anything",
+                         body: "About anything",
+                         user_id: "41b98f04-3ffd-11ec-9356-0242ac130003")
```

Comment:
Since all this data is at the topmost level, any specs added in the future will also have this declared even though they might not need it.

It's a good practice to keep the data creation as localized as possible. In this case, if we want to share the same data across validation specs, they can be nested in a `describe "Validations"`.